teaching-platfrom/controller.py

```diff
@@ -0,0 +1,41 @@
+from flask import Flask
+from flask import request
+import db_handler
+
+
+
+app = Flask(__name__)
+
+@app.route("/add", methods=["POST"])
```

Comment:
Consider implementing consistent routes. For example, for creating user you can use `/user` with post method instead of `/add` with post method.